emscripten.py

```diff
@@ -1907,6 +1907,18 @@ def debug_copy(src, dst):
         cmd += ['--export', export[1:]] # Strip the leading underscore
     shared.check_call(cmd)
 
+    if settings['DEBUG_LEVEL'] >= 2 or settings['PROFILING_FUNCS']:
+      if settings['DEBUG_LEVEL'] >= 4:
+        sourcemap_cmd = [shared.PYTHON, path_from_root('tools', 'wasm-sourcemap.py'), 
+                         base_wasm,
+                         '--dwarfdump=' + shared.LLVM_DWARFDUMP,
+                         '-o',  base_wasm + '.map']
+        if not settings['SOURCE_MAP_BASE']:
+          logging.warn("Wasm source map won't be usable in a browser without --source-map-base")
```

Comment:
notice that it might not be completely true if user chooses to use WebAssembly.instantiateStreaming or set SourceMap HTTP header.

Reply:
This message is copied from the current s2wasm path.  We can correct them both in a followup CL.